File: ArkhamAsylum.Lib/Models/Doctor.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace ArkhamAsylum.Lib.Models
{
    public class Doctor : Entity
    {
        public string Name { get; set; }
        public string FirstSurname { get; set; }
        public string SecondSutname { get; set; }
        public Guid AreaId { get; set; }
        [JsonIgnore]
        public Area Area { get; set; }
        [JsonIgnore]
        public ICollection<Diagnosis> Diagnoses { get; set; }
    }
}

File: ArkhamAsylum.Web/wwwroot/Scripts/Models/Doctor.js
class Doctor extends Entity
{
    constructor (json)
    {
        super (json)
        if (json)
        {
            this.Name = json.name;
            this.FirstSurname = json.firstSurname;
            this.SecondSurname = json.secondSurname;
            this.AreaId = json.areaId;
        }
        else
        {
            this.Name = "";
            this.FirstSurname = "";
            this.SecondSurname = "";
            this.AreaId = "00000000-0000-0000-0000-000000000000";
        }
    }
}